packages/styling/src/interfaces/ITheme.ts

```diff
@@ -6,10 +6,12 @@ export interface ITheme {
   palette: IPalette;
   fonts: IFontStyles;
   semanticColors: ISemanticColors;
+  isInverted: boolean;
```

Comment:
It might be nice for isInverted to be next to palette, so that it's a little more clear what's considered inverted.